themes/custom/az_barrio/templates/navigation/pager.html.twig

```diff
@@ -69,13 +88,20 @@
           {% endif %}
         </li>
       {% endfor %}
-      {# Add an ellipsis if there are further next pages. #}
+      {# Display an ellipsis if there are further next pages. #}
```

Comment:
Is it possible to remove the gray background on hover of the ellipses since they aren't clickable?

Reply:
@danahertzberg Thanks for the suggestion! I decided to remove the hover effect by deleting the span element with the `page-link` class around the ellipsis character. It doesn't seem appropriate for the ellipsis to have the `page-link` class and this also changes our theme override to be closer to Drupal core themes ([here is the Claro pager as an example](https://git.drupalcode.org/project/drupal/-/blob/11.x/core/themes/claro/templates/pager.html.twig#L72)).